app/data_models/questionnaire_store.py

```diff
@@ -27,6 +29,7 @@ def __init__(self, storage, version=None):
             self.collection_exercise_sid,
             version,
             self.submitted_at,
+            self.expires_at,
```

Comment:
Is this used? If we are always updating from the metadata, would we ever need to access it from questionnaire_state?

Reply:
So you think it's valid to be returned from get_user_data but just not set as a class attribute?

Reply:
Does it need to be returned by get_user_data, do we ever use it? I could only see it being called from the one place in questionnaire_store.py and tests

Reply:
Yeah not needed for the card so I removed it.